tests/conftest.py

```diff
@@ -234,3 +234,29 @@ def _generate_parser(entry_point_name):
         return ParserFactory(entry_point_name)
 
     return _generate_parser
+
+
+@pytest.fixture
+def generate_remote_data():
+    """Return a `RemoteData` node."""
+
+    def _generate_remote_data(computer, remote_path, entry_point_name=None):
+        """Return a `KpointsData` with a mesh of npoints in each direction."""
+        from aiida.common.links import LinkType
+        from aiida.orm import CalcJobNode, RemoteData
+        from aiida.plugins.entry_point import format_entry_point_string
+
+        entry_point = format_entry_point_string('aiida.calculations', entry_point_name)
+
+        remote = RemoteData(remote_path=remote_path)
+        remote.computer = computer
+
+        if entry_point_name is not None:
```

Comment:
More a question of style (if I write any logical statement like this). 
I've noticed elsewhere in the code that there is a tendency to go for the implicit 
``if entry_point_name:`` 
instead of checking explicitly for equality to ``None``. I realize in this case it will also be different while checked on an empty string though